azure/cpi.yml

```diff
@@ -69,7 +90,7 @@
 - type: replace
   path: /instance_groups/name=bosh/properties/azure?
   value: &cpi_conf
-    environment: AzureCloud
+    environment: ((environment))
```

Comment:
let's pull this override into a separate ops file so that default for most people continues to work. perhaps azure/custom-environment.yml